sos/report/plugins/scsi.py

```diff
@@ -50,7 +50,6 @@ def setup(self):
         self.add_cmd_output("lsscsi -i", suggest_filename="lsscsi")
 
         self.add_cmd_output([
-            "sg_map -x",
```

Comment:
Nack. Keep this, as it has been expressly requested by support teams. The fact we wrap this in a `timeout` command means we should still catch the timeout, and the deadlocked process would be killed when sos exits with the newly added `os._exit(0)`.